### ISSUE TITLE
Studio stopped opening on windows 10

### ISSUE THREAD

**Issue by rl00100** [SCIInstitute/ShapeWorks#1757]
Hi, 

I have previously been using ShapeWorks 6.1 on windows, I updated and was using 6.21 without issue. Today after i closed studio, I haven't been able to open it again. I have tried to fully uninstall all the packages and reinstall, but the issue is persistent.  The log is empty so i am struggling to spot where the issue is. 

Has this issue happened before? Are their some files that are persistent after uninstalling that could cause the issue, and i should delete (after reinstall studio still shows recent projects). 

Thanks, 

Rob

**Comment by akenmorris** [comment]
That's really strange.  We haven't heard of this happening before.  I can't think of any reason this would happen.  Have you tried rebooting?

If the Studio log files are empty, you could try looking at the system event viewer to see if perhaps there is some DLL problem preventing the program from starting.  I think it would be under the `application` logs.

**Comment by rl00100** [comment]
Thanks for the quick response. Yep I have tried rebooting, clearing temp files and App Data, and reinstall.

I am not used going through the event logs so I am not sure what the issue would be. I have attached a report file that was associated with the crash, in case you would be able to have a look and point me in the correct direction. 

[Report.txt](https://github.com/SCIInstitute/ShapeWorks/files/8636662/Report.txt)

**Comment by akenmorris** [comment]
Hmm, all it seem to say is that it was hanging.

Can you try running ShapeWorksStudio.exe from a command prompt?

cd to "C:\Program Files\ShapeWorks\bin"
ShapeWorksStudio.exe

Let me know if it prints anything to the terminal, or if it just hangs.

**Comment by rl00100** [comment]
That's how I have tried to run it. The terminal doesn't print anything, studio just hangs, and once it is closed the terminal just moves to a new line.

**Comment by akenmorris** [comment]
Does the `shapeworks.exe` executable work?

**Comment by rl00100** [comment]
Tested that in the terminal, shapeworks.exe prints the help as expected. I then ran studio and it suddenly started working.  Not sure what has changed and why it works now. I was going to ask how studio saves information about previous projects? As this stayed persistent, even with fresh installs, so i thought that there could be a corrupted settings/config file somewhere. 

Thanks for all the help, and fingers crossed it doesn't happen again.

**Comment by rl00100** [comment]
Working through the examples, the studio bug has popped up again. 
Error code printed into the terminal
```
General exception caught.
        Returncode: 3489660927
        Output: None
```

**Comment by akenmorris** [comment]
There are preferences stores.  On Windows, they should be under %APPDATA%

You're getting this error when running one of the use cases?  Is Studio popping up at all, or do you just see this at the end of the use case?

**Comment by rl00100** [comment]
This came after running the left-atrium use case i think. Studio popped up, but only as a white full window where it hanged.

**Comment by akenmorris** [comment]
And now it won't open at all anymore?

**Comment by rl00100** [comment]
Yes this is as far as it gets before hanging. Clicking on any project leads to a full crash 

![image](https://user-images.githubusercontent.com/25412285/167557287-8eb1b448-4c1f-486c-93e0-45a85bb8d73c.png)

**Comment by akenmorris** [comment]
Ah, ok.  And there are still no logs?

There is some chance the issue has been fixed since 6.2.1.  You could try the latest development release here:

https://github.com/SCIInstitute/ShapeWorks/releases/download/dev-windows/ShapeWorks-dev-6107-82e51564e-windows.exe

**Comment by rl00100** [comment]
Yep, still no logs. The issue stays persistent with different versions including the dev version. 
For now i have booted to linux to use the dev version and have no issues.
